packages/manager/modules/pci/src/projects/project/streams/stream/subscriptions/subscriptions.service.js

```diff
@@ -0,0 +1,99 @@
+import Subscription from './subscription.class';
+
+export default class PciProjectStreamStreamSubscriptionsService {
+  /* @ngInject */
+  constructor($q, OvhApiCloudProjectIo) {
+    this.$q = $q;
+    this.OvhApiCloudProjectIo = OvhApiCloudProjectIo;
+  }
+
+  getAll(projectId, streamId) {
+    return this.OvhApiCloudProjectIo
+      .Stream()
+      .Subscription()
+      .v6()
+      .query({
+        serviceName: projectId,
+        streamId,
+      })
+      .$promise
+      .then(subscriptions => this.$q.all(
```

Comment:
Couldn't be turned like this?

```suggestion
      .then(subscriptions => Promise.all(
```